### ISSUE TITLE
Installing modules with binary bindings

### ISSUE THREAD

**Issue by vsviridov** [hemanth/node-nightly#30]
Using `node-nightly` I get errors like `Error: Module version mismatch. Expected 51, got 48.` for modules with native bindings.

I tried doing something like `env NODE_PATH=/usr/local/lib/node_modules/node-nightly/node-nightly/ /usr/local/lib/node_modules/node-nightly/node-nightly/bin/npm install bcrypt` but that doesn't seem to help

**Comment by hemanth** [comment]
What your node version? Operating system?

**Comment by vsviridov** [comment]
OSX 10.10.5
Node 6.7.0

**Comment by linusnorton** [comment]
I'm also having this problem. Is it possible that npm is using the standard version of node and building dependencies against the wrong API? I'm not sure what the solution to this would be, maybe npm-nightly?

**Comment by vsviridov** [comment]
Yeah, even the npm bundled with `node-nightly` seems to just use the bindings in the default path.

**Comment by linusnorton** [comment]
Where is the npm bundled with node-nightly located?

**Comment by vsviridov** [comment]
On my mac it is here `/usr/local/lib/node_modules/node-nightly/node-nightly/bin/npm`

**Comment by linusnorton** [comment]
Weird, it doesn't seem to be in there or  `/usr/lib/node_modules/node-nightly/` on my linux box

**Comment by vsviridov** [comment]
I did `which node-nightly` and then followed the symlinks. Your node path likely going to be different.

**Comment by vsviridov** [comment]
Ok, so I ran the `node-nightly /usr/local/.../npm install` and it seemed to work.
